service/src/main/java/bio/terra/profile/service/profile/model/BillingProfile.java

```diff
@@ -22,7 +24,8 @@ public record BillingProfile(
     Optional<String> managedResourceGroupId,
     Instant createdTime,
     Instant lastModified,
-    String createdBy) {
+    String createdBy,
+    Optional<TpsPolicyInputs> policies) {
```

Comment:
I toyed with other options but ended up landing on adding this optional policies value to the `BillingProfile` record. The downside of this is that `BillingProfile` is no longer purely composed of information stored in BPM's DB -- we need to stitch in values from TPS to have a complete representation of the profile. 

Another option would be to introduce another record (e.g. `record BillingProfileDescription(BillingProfile profile, Optional<TpsPolicyInputs> policies)` and returning that from `getPolicies` and `listPolicies` instead. I'm undecided on whether this would be better or worse. One downside is it adds another class that represents a profile in BPM, but it does leave `BillingProfile` untouched. 

Open to input on this!

Reply:
I'd prefer to have a synthetic higher-level record and keep the database models pure. WSM does this with the[ WorkspaceDescription ](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/main/java/bio/terra/workspace/service/workspace/model/WorkspaceDescription.java)class, and then transforms _that_ into to the top level API response.